app/addons/index.js

```diff
@@ -1,10 +1,11 @@
 // native modules
-const fs = require('fs');
+const fs = require('fs-extra');
```

Comment:
this is not needed if you use Promise.promisifyAll(fs) etc.. (fs-extra is not even package.json at the moment)